Doc/library/sqlite3.rst

```diff
@@ -1223,6 +1250,42 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      *autocommit* has three allowed values:
+
+      * ``False``: select :pep:`249`-compliant transaction behaviour,
```

Comment:
```suggestion
      * ``False``: Select :pep:`249`-compliant transaction behaviour,
```

The others all use sentence case here, so presumably this should be no different.